cloud/linode/cilium_loadbalancers.go

```diff
@@ -481,6 +482,12 @@ func (l *loadbalancers) getCiliumLBIPPool(ctx context.Context, service *v1.Servi
 
 // NOTE: Cilium CRDs must be installed for this to work
 func (l *loadbalancers) ensureCiliumBGPPeeringPolicy(ctx context.Context) error {
+	if raw, ok := os.LookupEnv("BGPCustomIDMap"); ok {
```

Comment:
Can we add these two environment variables in readme here? https://github.com/linode/linode-cloud-controller-manager?tab=readme-ov-file#additional-environment-variables

Also, should we capitalize the environment variable to match with existing style?